docs/source/howto/ssh.rst

```diff
@@ -196,34 +196,67 @@ This section explains how to use the ``proxy_command`` feature of ``ssh`` in ord
 SSH configuration
 ^^^^^^^^^^^^^^^^^
 
-Edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing) and add the following lines::
+To decide whether to use the ``ProxyJump`` (recommended) or the ``ProxyCommand`` directive, please check the version of your SSH client first with ``ssh -V``.
+The ``ProxyJump`` directive has been added in version 7.3 of OpenSSH, hence if you are using an older version of SSH (on your machine or the *PROXY*) you have to use ``ProxyCommand``.
+
+To setup the proxy configuration with ``ProxyJump`, edit the ``~/.ssh/config`` file on the computer on which you installed AiiDA (or create it if missing)
+and add the following lines::
+
+  Host SHORTNAME_TARGET
+      Hostname FULLHOSTNAME_TARGET
+      User USER_TARGET
+      IdentityFile ~/.ssh/aiida
+      ProxyJump USER_PROXY@FULLHOSTNAME_PROXY
+
+  Host FULLHOSTNAME_PROXY
+      IdentityFile ~/.ssh/aiida
+
+replacing the ``..._TARGET`` and ``..._PROXY`` variables with the host/user names of the respective servers.
```

Comment:
```suggestion
replacing the ``..._TARGET`` and ``..._PROXY`` variables with the host/user names of the respective servers.

.. note :: 

    Please note that when using `ProxyJump` with AiiDA, the proxy and the target host must use the *same* `IdentityFile`.
    If the user directory of the proxy is shared with the target host (often the case in supercomputing facilities), this is ensured automatically.
```

Reply:
To be honest I think the formulation of proxy/target "using an `IdentityFile`" is a bit dangerous, people might start copying their private keys. Maybe something like: "the proxy and target must accept the same public key (matching the `IdentityFile`)?

Reply:
I see. Updated suggestion!

Reply:
the `IdentityFile` IS the private key